config/config_params.go

```diff
@@ -131,6 +131,8 @@ type Config struct {
 	Ipv6Support    bool `config:"bool;true"`
 	IgnoreLooseRPF bool `config:"bool;false"`
 
+	// FIXME Add this to libcalico-go
+	IptablesBackend                    string        `config:"oneof(legacy,nft);legacy;local"`
```

Comment:
If we think we might someday write a native nftables backend, do you think it would make sense to just use generic dataplane configuration?

e.g, `dataplane = iptables | ebpf | nftables`, but for now selecting `nftables` uses iptables in nft compat mode?

Reply:
I don't mind (can see this as a "top-level" mode or a sub-mode of "iptables" top-level mode) but I'd definitely prefer to keep `iptables-nft` and `nftables` distinct since they don't work well together (I think we'd need to maintain both modes if we do add nftables)